docs/docs/command-line-interface.mdx

```diff
@@ -419,6 +420,53 @@ This command is most commonly used to import old conversations into Rasa X to an
 them. Read more about [importing conversations into Rasa X](https://rasa.com/docs/rasa-x/installation-and-setup/deploy#1-import-existing-conversations-from-rasa-open-source).
 :::
 
+## rasa evaluate markers
+
+:::caution
+
+This feature is currently experimental and might change or be removed in the future. Share your feedback in the forum to help us make it production-ready.
+
+:::
+
+This command allows you to apply the [markers](./markers.mdx) you defined in your marker configuration file,
+to pre-existing dialogues stored in your [tracker store](./tracker-stores.mdx):
```

Comment:
I think it would be helpful if this also explains something about the expected output of the operation.